templates/base.html

```diff
@@ -3,7 +3,8 @@
 
 <head>
     <meta charset="utf-8">
-    <meta name="description" content="">
+    <meta name="description" content="boutique technology consultancy">
```

Comment:
Not a technical comment but I think we can write the `content` a bit more descriptive since this will be shown on the google search results page and in all link previews.

Reply:
make sense 👍